tbx/static_src/sass/components/_footer.scss

```diff
@@ -85,11 +85,14 @@
 
     &__logo {
         filter: grayscale(1) invert(1) brightness(0.52);
-        max-width: 70px;
 
         @include high-contrast-light-mode() {
             filter: none;
         }
+
+        &--narrow {
+            max-width: 70px;
```

Comment:
One final tweak I think would be nice - at the moment the EO logo now looks very big at the larger screen sizes - maybe a max-width of about 150px for the landscape logos would be good?

Reply:
Sure, added